packages/components/resource-pages/list-page.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from "react";
+import {
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageCreateLink,
+  ListPageFilter,
+  RowProps,
+  TableColumn,
+  TableData,
+  useActiveColumns,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable
+} from "@openshift-console/dynamic-plugin-sdk";
+import { K8sModel } from "@openshift-console/dynamic-plugin-sdk/lib/api/common-types";
+import { TFunction } from "i18next";
+import * as _ from "lodash";
+import { useTranslation } from "react-i18next";
+import { sortable } from "@patternfly/react-table";
+import { BucketClassType, NOOBAA_PROVIDER_MAP } from "../../constants";
+import {
+  NooBaaBackingStoreModel,
+  NooBaaBucketClassModel,
+  NooBaaNamespaceStoreModel
+} from "../../models";
+import { BucketClassKind, NamespaceStoreKind } from "../../types";
+import { referenceForModel } from "../../utils";
+import { OperandStatus } from "../../utils/generics/operand-status";
+import { Kebab } from "../../utils/kebab/kebab";
+import {
+  LaunchModal,
+  ModalMap,
+  useModalLauncher
+} from "../../utils/modals/modalLauncher";
+import { getName } from "../../utils/selectors/k8s";
+import "./list-page.scss";
+import { bucketClassResource, nameSpaceStoreResource } from "../resources";
+
+const tableColumnInfo = [
+  { className: "", id: "name" },
+  {
+    className: "",
+    id: "status"
+  },
+  {
+    className: "",
+    id: "provider"
+  },
+  {
+    className: "",
+    id: "buckets"
+  },
+  { className: "dropdown-kebab-pf pf-c-table__action", id: "" }
+];
+
+type ResourceTableProps = {
+  data: K8sResourceCommon[];
+  unfilteredData: K8sResourceCommon[];
+  loaded: boolean;
+  loadError: any;
```

Comment:
Can we be bit specific with the type of error?

Reply:
in the typescript type definition, it is mentioned as any, from there it is picking it up as any.

![image](https://user-images.githubusercontent.com/22158580/168281821-79c5cecd-08a2-4f85-a5a8-93a45e9c1b91.png)

we can make it string, but I think it should not be done until and unless we export the type as string only. if some other data type is received then there stays a chance of breaking.